src/pages/MergeReadiness/index.tsx

```diff
@@ -0,0 +1,349 @@
+            label={
+              <Text className="checkbox-label">
+                <FormattedMessage defaultMessage="I have provided an Ethereum address to my validator where I would like my fee rewards to be deposited." />
+              </Text>
+            }
+          />
+        </section>
+        <Heading level={3} className="mt50 mb30" id="additional-reminders">
+          <FormattedMessage defaultMessage="Additional reminders" />
+        </Heading>
+        <section>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="When Merge?" />
+          </Heading>
+          <Text className="mt20">
+            <FormattedMessage
+              defaultMessage="Public testnet progress is underway. When everything is safe and ready, the core developer teams will make this known, so stay tuned to the {blogLink} or other client team communication channels for the latest information."
+              values={{
+                blogLink: (
+                  <Link inline primary to="https://blog.ethereum.org">
+                    <FormattedMessage defaultMessage="Ethereum Foundation Blog" />
+                  </Link>
+                ),
+              }}
+            />
+          </Text>
+        </section>
+        <section>
+          <Heading level={3} id="testing-the-merge">
+            <FormattedMessage defaultMessage="#TestingTheMerge" />
+          </Heading>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="While waiting for the Mainnet transition to proof-of-stake, stakers are encouraged to participate in #TestingTheMerge. This is a great way to learn more about the Merge, practice going through it before Mainnet, and gain confidence in your setup." />
+          </Text>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="Kiln is the latest public testnet to have undergone its transition to proof-of-stake after undergoing a successful merge upgrade in March 2022. Kiln is open for anyone to interact with. Try sending some transactions, running a validator, or see if you can get slashed!" />
+          </Text>
+          <ul className="sub-checklist-item">
+            <li className="py5">
+              <Text>
+                <Link primary inline to="https://kiln.themerge.dev/">
+                  Kiln Homepage
+                </Link>
+                - Everything you need to get started with the network, including
+                a faucet
+              </Text>
+            </li>
+            <li className="py5">
+              <Text className="inline">
+                <Link
+                  primary
+                  inline
+                  to="https://github.com/remyroy/ethstaker/blob/main/merge-devnet.md"
+                >
+                  ETHStaker Kiln Guide
+                </Link>
+                - Step-by-step guide for how to spin up a Kiln node and
+                validator
+              </Text>
+            </li>
```

Comment:
- This guide is awesome and I set up my node with it as well! However, what do you think about providing a more "diverse" guide directory like https://notes.ethereum.org/@launchpad/kiln?
- But on the other hand, https://notes.ethereum.org/@launchpad/kiln was linked by https://kiln.themerge.dev, so probably not necessary.

Reply:
It's the next link listed here =)
<img width="658" alt="image" src="https://user-images.githubusercontent.com/54227730/170784111-99a8fe7b-6af5-4382-838c-54161ba77381.png">

Reply:
also just to draw your attention to this new page that helps folks to configure Geth to communicate with the CL https://geth.ethereum.org/docs/interface/merge

Reply:
Awesome, thanks @jmcook1186... Any idea if the other EL clients have similar documentation? Been trying to present the info evenly for all of them if possible, to help level the playing field for client diversity.